services/service.go

```diff
@@ -265,6 +303,26 @@ func (s *Service) HandleSurveyDDTRequestTerms(sdq *SurveyDDTRequest) (network.Me
 
 // HandleSurveyKSRequest handles the reception of the aggregate local result to be key switched
 func (s *Service) HandleSurveyKSRequest(skr *SurveyKSRequest) (network.Message, error) {
+	// sanitize params
+	if err := emptySurveyID(skr.SurveyID); err != nil {
+		log.Error(err)
```

Comment:
Usually the error is logged in `onet`, so no need to log it here, too...

Reply:
@ineiti but it would be probably too difficult to trace back it's source no? e.g. line of code. In the end does it make a difference if it's logged in two sides?

Reply:
Halfway true ;) The latest onet expects the following:
```
return nil, xerrors.Errorf("survey id is empty: %+v", err)
```
Then it will correctly output the line-#.

Reply:
@ineiti I fixed the double logging in the latest commit 3fb3422. Is it ok like this?